### DISCUSSION TITLE
Slider Sample

### DISCUSSION THREAD

**Discussion by jwestes** [RudolphRiedel/FT800-FT813#71] [Q&A] [answered]
Do you have a sample that shows how the slider works?  I found one here:
https://os.mbed.com/users/dreschpe/code/FT800_RGB_demo/file/07f4e7d61702/main.cpp/

I have the slider working, but nothing happens when I touch the slider.  I suspect the value is changed back on the next redraw of the screen, so I'm not seeing the change, but I don't know how to work around that.  Here is the code I have to draw and track the slider.
    EVE_cmd_dl_burst(TAG(2));
    EVE_cmd_slider_burst(right - sliderWidth - margin, nextTop + (0 * lineSpacing) + 9, sliderWidth, sliderHeight, 0, surfDevice->MaxExtendTime() * 10, 100);
    EVE_cmd_track(right - sliderWidth - margin, nextTop + (0 * lineSpacing) + 9, sliderWidth, sliderHeight, 2);

**Comment by RudolphRiedel** [comment, marked as answer]
I do not have a full example, I maybe should add one to my demo since this is something less obvious.
I posted this before, somewhere, perhaps on the BRT forum...

In my last project I was using this in TFT_display():
EVE_cmd_dl_burst(TAG(30));
EVE_color_rgb_burst(LIGHT_GREY);
EVE_cmd_slider_burst(50, 270, 700, 20, 0, rpm_request, 550);

And then this in TFT_touch:
switch(tag)
{
    case 0:
        if(touchtest == 0xffffffff) /* display no longer touched */
        {
            toggle_lock = 0;

            if(track != 0 )
            {
                EVE_cmd_track(0, 0, 0, 0, 0); // stop tracking
                track = 0;
            }
        }
        break;
...
    case 30:
        if(track == 0)
        {
            if(toggle_lock == 0 && active_nad != 0)
            {
                EVE_cmd_track(20,240,790,70,30); /* start tracking */
                track = 10;
                toggle_lock = 42;
            }
        }
        else
        {
            uint32_t tracker_val = EVE_memRead32(REG_TRACKER);
            if((tracker_val & 0xff) == 30)
            {
                tracker_val = tracker_val / 7809031; /* -> 550 max */
                rpm_request = tracker_val;
            }
        }
        break;
...

**Comment by jwestes** [comment]
Thanks!  It took me a while to get this integrated into the framework I've built around this library, but the sliders now work.
